Sources/Plasma/CoreLib/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(CoreLib_SOURCES
     plLoadMask.cpp
     plProduct.cpp
     plViewTransform.cpp
-    hsWindows.cpp
+    $<$<PLATFORM_ID:Windows>:hsWindows.cpp>
```

Comment:
In that case, the `#ifdef HS_BUILD_FOR_WIN32` ... `#endif` should be removed from hsWindows.cpp.